Lib/typing.py

```diff
@@ -2384,12 +1592,10 @@ class io:
 io.__name__ = __name__ + '.io'
 sys.modules[io.__name__] = io
 
-
-Pattern = _TypeAlias('Pattern', AnyStr, type(stdlib_re.compile('')),
-                     lambda p: p.pattern)
-Match = _TypeAlias('Match', AnyStr, type(stdlib_re.match('', '')),
-                   lambda m: m.re.pattern)
-
+Pattern = _GenericAlias(type(stdlib_re.compile('')), AnyStr,
```

Comment:
You can use `stdlib_re.Pattern` instead of `type(stdlib_re.compile(''))` and `stdlib_re.Match` instead of `type(stdlib_re.match('', ''))`.